src/edgepi/pwm/edgepi_pwm.py

```diff
@@ -46,14 +61,16 @@ def get_frequency(self):
         """
         return self.pwm.get_frequency_pwm()
 
-    def set_duty_cycle(self, duty_cycle: float):
+    def set_duty_cycle(self, duty_cycle: int):
         """
         Set duty_cycle
         Args:
             duty_cycle (int): duty_cycle value
         Returns:
             N/A
         """
+        self.__check_range(duty_cycle, PWM_MIN_DUTY_CYCLE, PWM_MAX_DUTY_CYCLE)
+        duty_cycle = duty_cycle/100.0
```

Comment:
avoid reusing variables. it is a bad practice : )
 
many sources on this. here is one for c++ -> http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es26-dont-use-a-variable-for-two-unrelated-purposes

let me elaborate on this a bit even though it may not be a big deal at all. when I see the doc string like this `duty_cycle (int): duty_cycle value` and then the call like this `self.pwm.set_duty_cycle_pwm(duty_cycle)` my assumption i that you are sending the value that you get from the user to `pwm`. but this is not the case. you are manipulating that number. it is very obvious in this function what is happening. but imagine if you have a big function, the reader of the code might easily not notice the changes. 

if feeling lazy just do something like this: 
```
self.pwm.set_duty_cycle_pwm(duty_cycle/100.0)
```

Reply:
ad932314ef22a561d9dcf868a506e860dec2d76a

doc string edited and added suggested fix